acceptance/managers/run_combination.go

```diff
@@ -0,0 +1,189 @@
+// +build acceptance
+
+package managers
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	"github.com/buildpacks/pack/internal/style"
+)
+
+type ComboValue int
+
+const (
+	Current ComboValue = iota
+	Previous
+	DefaultKind
+)
+
+func (v ComboValue) String() string {
+	switch v {
+	case Current:
+		return "current"
+	case Previous:
+		return "previous"
+	case DefaultKind:
+		return "default"
+	default:
+		return ""
+	}
+}
+
+var defaultRunCombo = []*RunCombo{
+	{Pack: Current, PackCreateBuilder: Current, Lifecycle: DefaultKind},
+}
+
+type RunCombo struct {
```

Comment:
For clarity, perhaps we could have the `RunCombo` typedef first, then `defaultRunCombo`, then `ComboValue`? Or do you want to connect `RunCombo` with its `func`s?

Reply:
That was the goal. Happy to move it above `defaultRunCombo` but I'd probably rather keep all of the `ComboValue` bits together.